tests/functional/cypress/specs/floating-menu-buttons-form-design.cy.js

```diff
@@ -12,22 +12,30 @@ Cypress.Commands.add('waitForLoad', () => {
 
 describe('Form Designer', () => {
   beforeEach(()=>{
+    debugger;
     cy.viewport(1000, 1800);
     cy.waitForLoad();
     cy.kcLogout();
```

Comment:
Is there a particular reason why cy.kcLogout gets removed everywhere else but not here?

Reply:
@jasonchung1871 This file is not working anymore, so please ignore this. I created a new folder **e2e**. I moved all the files I updated /created to that folder. Cypress tests will run from that folder. Could you please check that?